CODEOWNERS

```diff
@@ -281,6 +281,8 @@ build.json @home-assistant/supervisor
 /tests/components/dsmr/ @Robbie1221 @frenck
 /homeassistant/components/dsmr_reader/ @depl0y @glodenox
 /tests/components/dsmr_reader/ @depl0y @glodenox
+/homeassistant/components/duckdns/ @bkbilly
+/tests/components/duckdns/ @bkbilly
```

Comment:
To be clear:
While new code owners are very welcome, only add yourself here if you plan to actively work on the DuckDNS integration by solving issues, reviewing PRs and so on going forward. If you're not planning to do that, don't add yourself.

Reply:
There wasn't anyone selected as a code owner, so I thought of putting myself. I will be happy to help out with any issues that might show up.